superchain/configs/rehearsal-0-bn/rehearsal-0-bn-0.toml

```diff
@@ -0,0 +1,70 @@
+name = "rehearsal-0-bn-0"
+public_rpc = ""
+sequencer_rpc = ""
+explorer = ""
+superchain_level = 0
+governed_by_optimism = false
+superchain_time = 0
+data_availability_type = "eth-da"
+chain_id = 420120009
+batch_inbox_addr = "0x00aA742D2EEf0B65Eb1dC7633cbAf8Ef24A945B3"
+block_time = 2
+seq_window_size = 3600
+max_sequencer_drift = 600
+
+[hardforks]
+  canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  holocene_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  isthmus_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
```

Comment:
Should we manually add the `interop_time` here?

Reply:
I went ahead and added the timestamp to superchain.toml and then ran `just apply-hardforks`. I think we want that timestamp in the configs so that it will be included when we update op-geth's import of superchain-registry to include these configs. Then we can test op-nodes running with this timestamp

Reply:
We may want to cover this in the readme instructions for `import_devnet`.